src/components/charts/DistributionPlot/distPlotD3.js

```diff
@@ -109,22 +109,28 @@ export class CdfChartD3 {
         `translate(${this.calc.chartLeftMargin}, ${this.calc.chartTopMargin})`,
       );
 
+    const common = this.getCommonThings();
     if (this.hasDate('continuous')) {
-      const distributionChart = this.addDistributionChart();
-      if (this.hasDate('discrete')) {
-        this.addLollipopsChart(distributionChart);
-      }
+      this.addDistributionChart(common);
+    }
+    if (this.hasDate('discrete')) {
+      this.addLollipopsChart(common);
     }
     return this;
   }
 
-  addDistributionChart() {
-    const areaColorRange = d3.scaleOrdinal().range(this.attrs.areaColors);
-    const dataPoints = [this.getDataPoints('continuous')];
-
+  /**
+   * @returns {*}
+   */
+  getCommonThings() {
```

Comment:
can you use a better name? "Things" isn't very specific. Maybe, "getXandYDomainParameters" or something?